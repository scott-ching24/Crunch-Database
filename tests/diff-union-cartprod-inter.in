diff a 
union a
cartprod a
inter a
set a 1 1 3 3 5 4 4
diff a
union a
cartprod a
inter a
set b 1 1 3 4 5 7 7 6 8
diff a b
union a b
cartprod a b
inter a b
bye
