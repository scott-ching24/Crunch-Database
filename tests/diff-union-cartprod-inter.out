> invalid command

> invalid command

> invalid command

> invalid command

> ok

> invalid command

> invalid command

> invalid command

> invalid command

> ok

> [6 7 8]

> [1 3 4 5 6 7 8]

> [ [1 1] [1 1] [1 3] [1 4] [1 5] [1 7] [1 7] [1 6] [1 8] [1 1] [1 1] [1 3] [1 4] [1 5] [1 7] [1 7] [1 6] [1 8] [3 1] [3 1] [3 3] [3 4] [3 5] [3 7] [3 7] [3 6] [3 8] [3 1] [3 1] [3 3] [3 4] [3 5] [3 7] [3 7] [3 6] [3 8] [5 1] [5 1] [5 3] [5 4] [5 5] [5 7] [5 7] [5 6] [5 8] [4 1] [4 1] [4 3] [4 4] [4 5] [4 7] [4 7] [4 6] [4 8] [4 1] [4 1] [4 3] [4 4] [4 5] [4 7] [4 7] [4 6] [4 8] ]

> [1 3 4 5]

> bye
